end-to-end/e2e.spec.ts

```diff
@@ -40,6 +40,38 @@ describe('e2e', function () {
     await contracts.uniswapPCVDeposit.deposit()
   })
 
+  describe('PCV Equity Minter + LBP', async function() {
```

Comment:
Moved because of an order of operation dependency. CR oracle changes mess this up